datawig/imputer.py

```diff
@@ -85,6 +87,12 @@ def __init__(self,
         self.precision_recall_curves = {}
         self.calibration_info = {}
 
+        self.__class_patterns = None
+        self.is_explainable = np.any([isinstance(encoder, CategoricalEncoder) or isinstance(encoder, TfIdfEncoder)
```

Comment:
I don't think there are obvious extensions to other input encoders, maybe to numerical. I don't think we will do this anytime soon, since it's not something we see in applications a lot.